kickstart/PXELinux_rhel_atomic.erb

```diff
@@ -0,0 +1,14 @@
+<%#
+kind: PXELinux
+name: RHEL Atomic PXE
```

Comment:
Right, but then I'd have to define a new OS ("Atomic") which would encompass CentOS/Fedora/RHEL Atomic on core, would that be preferred?